src/tmp_fetch_send_json.js

```diff
@@ -0,0 +1,31 @@
+// Temporary workaround for returning JSON payloads for non-200 (failed) responses
+// Remove this file when this fix is implemented in `fetch-send-json`
+
+let fetch = (typeof window === 'undefined' || window.name == 'nodejs') ? require('node-fetch') : window.fetch;
```

Comment:
this came from `fetch-send-json`, so I'm not sure if this is intentional -- when I find out who is responsible for that package I intend to make a PR against it to add these changes for JSON responses to non-200 error code responses; I'll includes this change there to see if the author had any intention when they made it == rather than ===....

Reply:
moved to new `near` org, but feel free to make changes to package itself:

https://github.com/near/send-json

` window.name == 'nodejs'` is likely some legacy copy-pasta I don't even know in  which cases is needed (maybe simulated Jest `window` or smth like that)